### ISSUE TITLE
Install step fails "silently". 

### ISSUE THREAD

**Issue by rewolff** [esden/summon-arm-toolchain#27]
I jumped in and typed "./summon-arm-toolchain" . It compiled a whole lot of things, and in the end I was left with.... nothing. 

It turns out (I think) that it compiles and then installs the programs, but it fails silently (burried under the tons of "normal" output) at installing because I had not added "SUDO=sudo" on the commandline. 

I think the compile step should create a stamp called "compile", and then the install should stamp "install", but only when succesful. 

And in the end, the cleanup should only delete the build when the install is successful .

**Comment by twaldecker** [comment]
Hi rewolff,

you should read the README File:

## PREFIX=

By default the installation prefix is "$(HOME)/sat" you can change it to "/usr" then the binaries will be installed into "/usr/bin" and the toolchain specific
files in "/usr/arm-none-eabi" assuming you did not change the TARGET variable.

**Comment by rewolff** [comment]
You might be right. On the other hand, if I already set "prefix=/usr", then I think I'll have to wait out a whole new compile session because my suggestion "stamp-compile" hasn't been implemented.

**Comment by twaldecker** [comment]
If I understand you right, you want a new stamp. Just go a head fork it. Seems not too difficult to implement.

**Comment by esden** [comment]
The default directory and intended usage of this script is to install in your home directory. If you use non standard directory then using sudo for installing in root owned directories would be useful. Patch for this functionality is welcomed. :)

**Comment by rewolff** [comment]
When I started developing for x86 I typed: "apt-get install gcc" and I had a compiler. substitute avr-gcc and I get the compiler for AVR. It should be this simple for ARM as well. And apparently it is. It seems I now have two ARM compilers installed, one through the normal APT system. I'm happy. 

In hindsight, I think I found the installed compiler in ~/sat a few months later. It silently installed it there, and didn't tell me about it. Or, as had been said before I should've read the README.
